metal/resource_metal_device.go

```diff
@@ -353,6 +353,21 @@ func resourceMetalDevice() *schema.Resource {
 				Default:     false,
 				ForceNew:    false,
 			},
+			"termination_time": {
+				Type:        schema.TypeString,
+				Description: "Timestamp for device termination, use with care. For example \"2021-09-03T16:32:00+03:00\". If you don't supply timezone info, timestamp is assumed to be in UTC.",
+				Optional:    true,
+				ForceNew:    false,
+				Deprecated:  "You probably don't want to use this attribute, we added it for sweeping test artifacts.",
```

Comment:
I don't think we need to be defensive or prescriptive about this field.  The API exposes it so users may consume it however they like.

For example:

```hcl
variable "temporary_hours" { default = 14*24 } 
variable "expires" { default = timeadd(formatdate("YYYY-MM-01T00:00:00-00:00"), "${var.temp_hours}h") }
variable "temp_count" { default = 3 }

// add 3 temporary servers for extra work needed at the beginning of the month
resource "metal_device" "temporary" {
  count = var.expires > timestamp() ? var.temp_count : 0
  termination_time =  var.expires
}
```

Reply:
Alright!